compiler/resolution/functionResolution.cpp

```diff
@@ -7715,6 +7748,18 @@ static Type* moveDetermineRhsType(CallExpr* call) {
     }
   }
 
+  // TODO do we still need this
```

Comment:
Are you planning to check on this before merging?

Reply:
I throw an assertion in that block and will run another gasnet testing with that. If it is never triggered, I'll remove the block.

Reply:
Yep, that block can go away.